Source/Core/AudioCommon/AudioDevice.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+struct AudioDevice
+{
+
+public:
+	AudioDevice() {};
+	AudioDevice(u32 _index, std::string _name, std::string _path, std::string _id) : index(_index), name(_name), path(_path), id(_id) {};
+
+	inline bool operator==(const AudioDevice& other) { return id == other.id; }
+	inline bool operator!=(const AudioDevice& other) { return !operator==(other); }
```

Comment:
inline isn't necessary for methods defined in a class body. They're inline by default.